src/js/view/components/nextuptooltip.js

```diff
@@ -152,6 +152,7 @@ define([
             this._related = this._api.getPlugin('related');
             // Only switch to related mode if there is a related playlist
             this._related.on('playlist', this.onRelatedPlaylist.bind(this));
+            this._related.on('cached', this.onRelatedCached.bind(this));
```

Comment:
If the cached event means the playlist hasn't changed, you can handle the events similarly:
`this._related.on('playlist cached', this.onRelatedPlaylist.bind(this));`